src/App.php

```diff
@@ -157,7 +165,7 @@ public function initLayout($layout, $options = [])
         }
         $layout->app = $this;
 
-        $this->html = new View(['defaultTemplate'=>'html.html']);
+        $this->html = new View(['defaultTemplate' => $this->defaultTemplate]);
```

Comment:
at best we can make it possible for someone to inject "html" property, so 

if (!$this->html) {
 ...
}